tests/ApolloExtensionsClient.test.ts

```diff
@@ -1,5 +1,5 @@
+import { times } from "lodash-es";
```

Comment:
Is there a reason sometimes we rename to `_{method}` and sometimes we don't? Should we standardize on a convention?

Reply:
I only prefixed it because of a scope conflict for filter I believe